src/pl/plr.spec

```diff
@@ -3,10 +3,14 @@ Name:           plr-hawq
 Version:        08.03.00.14
 Release:		0
 Prefix:         /usr/local
-License:		GPL
+License:		Apache
+BuildRequires:  R-devel >= 3.1.0
```

Comment:
I would prefer: R-devel = 3.1.0 and R = 3.1.0. The reason is that we only certified plr in hawq with R 3.1.0, we don't know if there are any compatible issue R > 3.1.0. Any thought?